web/src/components/vis/VisPca.vue

```diff
@@ -1,10 +1,24 @@
 <template>
-  <svg ref="svg" :width="width" :height="height" xmlns="http://www.w3.org/2000/svg">
+  <svg
+    ref="svg"
+    :width="width"
+    :height="height"
+    xmlns="http://www.w3.org/2000/svg"
+  >
     <g class="master">
-      <g class="axes"></g>
-      <g class="plot"></g>
-      <g class="ellipse" transform="translate(0, 0) rotate(0) scale(1, 1)">
-        <circle cx="0" cy="0" r="1" style="fill: none; stroke: black;" vector-effect="non-scaling-stroke" />
+      <g class="axes" />
+      <g class="plot" />
+      <g
+        class="ellipse"
+        transform="translate(0, 0) rotate(0) scale(1, 1)"
+      >
+        <circle
```

Comment:
I'm happy to adopt this style in writing templates, but is this actually more legible than the single line version it replaced?

Reply:
I believe >100 chars was the reason for breaking it, not legibility.   Single line version is fine for <100 characters.

Reply:
Probably worth mentioning that the rest of the project uses pug, which is what I've configured the linter for.

I made that call without asking anyone (sorry), but we should probably be consistent for the future.  If you're not happy with pug as a template language, we can move everything back to html.  Zach and I (and maybe others?) like it a lot, but @matthewma prefers plain html, so it's not a unanimous thing.

Reply:
Ah I can redo this in Pug.

I'm a big fan of it actually, but when I did the Vue tutorials I also liked using HTML templates.

Reply:
I think we've got ourselves a rule conflict: the linter wants all attributes on a single line, but also wants to enforce a max line length. For my `circle` element, I have to break one rule or the other. I thought I'd be able to break the line between the attributes (due to the `multiLineSeparator` being specified) but it doesn't seem to be working. I'll push the thing that's breaking and maybe someone can take a look at it.

Reply:
Just needs to change the indent.  line overhang like that requires the overhang to be twice indented from the first line.

Reply:
See b57a7f9

Reply:
Ah gotcha, thanks.